src/xdocs/config_metrics.xml

```diff
@@ -111,17 +128,52 @@
   &lt;property name=&quot;max&quot; value=&quot;7&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Code Example:</p>
+          <source>
+public class Test
+{
+  public static void main(String ... args)
+  {
+    boolean a = true;
+    boolean b = false;
+
+    boolean c = (a &amp; b) | (b &amp; a) ^ (a &amp; b) | (b &amp; a);
+    //OK, since the complexity here is 7(max allowed 7) as (&amp;,|,^) are counted as 1.
+
+    boolean d = (a | b) ^ (a | b) ^ (a || b) | (b &amp; a) ^ b;
+    //violation,complexity here is 8 (max allowed 7) as (|,^,&amp;,||) are counted as 1.
+  }
+}
+          </source>
 
         <p>
           To configure the check to ignore <code>&#x26;</code> and
           <code>|</code>:
         </p>
+
         <source>
 &lt;module name="BooleanExpressionComplexity"&gt;
   &lt;property name="tokens" value="BXOR,LAND,LOR"/&gt;
 &lt;/module&gt;
-        </source>
-
+      </source>
```

Comment:
`</source>` tag looks to not be indented as far over as it was before create an unnecessary change. The other tags should align with this tag too.

Reply:
Done